addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: dex-k8s-authenticator
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: dex-k8s-authenticator
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.1-1"
```

Comment:
revision needs to be bumped

Reply:
It was - in master. This PR just cherry-picks the squashed commit.

Reply:
Went from 1.2.0-1 (in dex-k8s-authenticator-1.yaml) to 1.2.1-1

Reply:
ok.